src/main/java/teammates/logic/api/Logic.java

```diff
@@ -834,11 +834,11 @@ public StudentProfileAttributes getStudentProfile(String googleId) {
         return studentsLogic.getStudentsForCourse(courseId);
     }
 
-     /**
-     * Preconditions: <br>
-     * * All parameters are non-null.
-     * @return Empty list if none found.
-     */
+    /**
+    * Preconditions: <br>
+    * * All parameters are non-null.
+    * @return Empty list if none found.
+    */
```

Comment:
Only the very first line should be moved. The Javadoc block now looks:
```java
/**
* content
*/
```
where it should be:
```java
/**
 * content
 */
```